src/makeRobot.js

```diff
@@ -38,7 +38,60 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    get info() {
+      return `name: ${
+        this.name
+      }, chip version: ${
+        this.version
+      }, wheels: ${this.wheels}`;
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    coords: {
```

Comment:
добавляй відступ між методами
```suggestion
    version,
    
    get info() {
      return `name: ${
        this.name
      }, chip version: ${
        this.version
      }, wheels: ${this.wheels}`;
    },
    
    get location() {
      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
    },
    
    coords: {
```